Raven.Database/Storage/Esent/SchemaCreator.cs

```diff
@@ -815,7 +815,7 @@ public void CreateListsTable(JET_DBID dbid)
                     cbKeyMost = SystemParameters.KeyMost,
                     cbVarSegMac = SystemParameters.KeyMost,
                     szKey = by_name_and_etag,
-                    grbit = CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,
+                    grbit = CreateIndexGrbit.None,
```

Comment:
In schema 5.4 we had `CreateIndexGrbit.IndexDisallowNull` here - I think we should have the same now